en/docs/learn/references/natural-functions.md

```diff
@@ -20,14 +20,14 @@ Follow the steps below to implement the integration.
 ### Step 2: Define Types
```

Comment:
L15 Can we use a more relevant name?

Reply:
`Generate type definitions for natural Function`
Wdyt?

Reply:
This was for L15 - project name.

Reply:
Should also be changed as

Enter `ProjectName` as the project name.

Reply:
Updated